Sources/BraintreeSEPADirectDebit/SEPADirectDebitAPI.swift

```diff
@@ -8,9 +8,9 @@ class SEPADirectDebitAPI {
     
     private let apiClient: BTAPIClient
     
-    @objc(initWithAPIClient:)
-    init(apiClient: BTAPIClient) {
-        self.apiClient = apiClient
+    @objc(initWithAuthorization:)
```

Comment:
We can remove this `@objc` since this entire class is internal - technically not part of this work but fine to happen in this PR.